src/index.ts

```diff
@@ -179,6 +249,33 @@ export class EthrDID {
     return receipt.transactionHash
   }
 
+  async createSetAttributeHash(attrName: string, attrValue: string, exp: number) {
+    if (typeof this.controller === 'undefined') {
+      throw new Error('a web3 provider configuration is needed for network operations')
+    }
+    return this.controller.createSetAttributeHash(attrName, attrValue, exp)
+  }
+
+  async setAttributeSigned(
+    key: string,
+    value: string | Uint8Array,
+    expiresIn = 86400,
+    signature: MetaSignature,
+    gasLimit?: number,
```

Comment:
Let's remove the `gasLimit` parameter. It should be extracted from the `txOptions`
```suggestion
```